master/getting-started/kubernetes/index.md

```diff
@@ -1,41 +1,195 @@
 ---
-title: Calico for Kubernetes
+title: Quickstart for Calico on Kubernetes
 ---
 
-Calico enables networking and network policy in Kubernetes clusters across the cloud.  Calico works
-everywhere - on all major public cloud providers and private cloud as well.
 
-Calico uses a pure IP networking fabric to provide high performance networking, and its battle-tested policy engine
-enforces high-level, intent-focused network policy.  Together, Calico and Kubernetes provide a secure,
-cloud-native platform that can scale your infrastructure to hundreds of thousands of workloads.
+### Overview
 
-## Installing Calico for Kubernetes
+This quickstart gets you a single-host Kubernetes cluster with Calico
+in approximately 15 minutes. You can use this cluster for testing and
+development.
 
-There are a number of ways to install Calico and Kubernetes.  The [installation documentation](installation)
-includes links to a number of popular guides and installers which use Calico. It also
-includes information on installing Calico on a from-scratch Kubernetes cluster using either a self-hosted Kubernetes manifest,
-or by integrating Calico into your own configuration management scripts.
+To deploy a cluster suitable for production, refer to [Installation](/getting-started/kubernetes/installation/).
 
-## Using Calico with Kubernetes
 
-Once you have a Kubernetes cluster with Calico installed, the following articles will help you
-get familiar with Calico and make the most of the features that Calico provides.
+### Requirements
```

Comment:
Might want to word this as "This guide is known to work on systems which meet the following requirements" - I don't know we can / want to say it will ONLY work for these. Maybe not though, just a thought.

Reply:
In my experience, it literally does not work (reproduceably) if they don't meet these requirements. I can say that the disk space might be a little high but definitely, the others...it literally just doesn't work without these.